src/main/kotlin/org/rust/ide/intentions/AddCurlyBracesIntention.kt

```diff
@@ -49,11 +49,11 @@ class AddCurlyBracesIntention : RsElementBaseIntentionAction<AddCurlyBracesInten
     }
 
     override fun invoke(project: Project, editor: Editor, ctx: Context) {
-        val identifier = ctx.path.referenceNameElement
+        val identifier = ctx.path.referenceNameElement?.text ?: ""
 
         // Create a new use item that contains a glob list that we can use.
         // Then extract from it the glob list and the double colon.
-        val newUseSpeck = RsPsiFactory(project).createUseSpeck("dummy::{${identifier.text}}")
+        val newUseSpeck = RsPsiFactory(project).createUseSpeck("dummy::{$identifier}")
```

Comment:
It would be great to add a test that checks that we provide this intention in the following case: `use foo::/*caret*/`.
It's a definitely improvement of the intention and it would be great to commit such behavior not to break it in the future

Reply:
Not related to this PR